docs/usage.rst

```diff
@@ -408,6 +408,15 @@ and a python wheel, it is possible to test for the variable ``SKBUILD``:
       message(STATUS "The project is built using scikit-build")
     endif()
 
+
+How does scikit-build know where to find compiled modules?
+----------------------------------------------------------
+
+Compiled targets and other install files are identified by parsing the CMake-generated
+``install_manifest.txt`` file from :func:`skbuild.constants.CMAKE_BUILD_DIR()`. Note:
+if scikit-build is driving nested CMake projects (externalproject / "superbuild"), the
+manifest file may not be present in the ``CMAKE_BUILD_DIR``.
```

Comment:
The way to currently deal with this is to explicitly include the `cmake_install.cmake` by adding code like the following in the outer project:

```
#-----------------------------------------------------------------------------
# Install inner project
message(STATUS "Adding install rules for inner project")
install(CODE "
include(\"${CMAKE_BINARY_DIR}/inner-build/cmake_install.cmake\")
")
```

Notes:
* Make sure to tweak `${CMAKE_BINARY_DIR}/inner-build` based on your project.
* It may be possible to generalize this to include `cmake_install.cmake` from other projects also built using `ExternalProject` ...